gradle/publish.gradle

```diff
@@ -27,6 +18,50 @@ This file should contain:
 - gradle.publish.secret: the secret to publish the plugin to the Gradle Plugins Repository
 		''')
     }
+
+    cli {
+        using(project)
+    }
+
+    bintray {
+        def bintrayCredentials = {
+            boolean isDryRun = cli['dryRun'].or(true).boolean
+            return isDryRun ?
+                    ['bintrayRepo': 'n/a', 'bintrayUser': 'n/a', 'bintrayKey': 'n/a'] :
+                    new File("${System.getenv('BINTRAY_PROPERTIES')}")
```

Comment:
Have you defined the CI job to set this var up?

Reply:
yep, this is configured:
https://ci.novoda.com/job/gradle-static-analysis-plugin-snapshot/configure